playwright/navigation.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { expect, test } from '@playwright/test';
+
+test('Check content', async ({ page }) => {
+  await page.goto('/');
+  await expect(page.getByTestId('intro')).toBeVisible();
```

Comment:
let's also test clicking on an element

Reply:
Updated the test. Now test runner clicks on the links and checks if the URL has changed.

[da3e704](https://github.com/upleveled/next-portfolio-dev/pull/87/commits/da3e704f4267f6aead4142e5cb4052c94b5647f9)

Reply:
Added a new assertion to check if the click scrolled down
[73981b4](https://github.com/upleveled/next-portfolio-dev/pull/87/commits/73981b4b2a64b0a0dd84356ec38d6f995982e6d7)